R/env.R

```diff
@@ -552,6 +595,26 @@ env_clone <- function(x, parent = env_parent(x)) {
   list2env(as.list(x, all.names = TRUE), parent = parent)
 }
 
+#' Does environment inherits from another environment?
```

Comment:
This is not consistent with how we have written titles so far, e.g. all those "Is object an xxx" titles for the type predicates. Should we adopt this style from now on?